test/integration/model/scope.test.js

```diff
@@ -108,5 +108,37 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           return this.ScopeMe.scope('issue8473').findAll();
         });
     });
+
+    it('should be applied before query options for attributes exclude', function() {
```

Comment:
Its worth adding a test for case when `scope` is trying to exclude some attribute but actual call is trying to include it. Which again comes to a discussion matter, I think scope should still exclude that attribute

Reply:
@sushantdhiman The test below this one does that (`should be applied before query options for attributes include`)

Reply:
`lowAccess` is only setting attributes to `['other_value', 'access_level'],`, its not excluding `email`, so shouldn't `include` at method level should be able to inject `email` to attributes?

I think it would make sense if scope say `exclude` something then finder method should not be allowed to include that attribute, if scopes are just specifying attributes or using `include`, then finder method should be able to expand that list of attributes

Reply:
@sushantdhiman I think I see what you're asking for, I'll add that test but it does function like that.